src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
```

Comment:
```suggestion
## v0.9.0
```

Reply:
I think each minor release should also have a release note.

Reply:
@Jaanki we discussed this over Slack (https://kubernetes.slack.com/archives/C010RJV694M/p1614777461479900) and the consensus seems to be that we don't want to announce each dev milestone release as a full-blown release. So we can keep this PR as a running draft for the 0.9.0 cycle, and publish together with the final 0.9.0 release. 

I also agree that we should focus on notebale user-facing changes only (and don't document each and every change), to be consistent with previous releases.

Reply:
We can use the latest version once we are happy to update the notes here:

https://github.com/submariner-io/releases/releases/tag/v0.9.0-m1

Reply:
If there are lower-level, internal code changes we want to document, we can add it to the specific repo's release tag notes.